src/app/services/technical-record/technical-record.service.ts

```diff
@@ -143,9 +143,16 @@ export class TechnicalRecordService {
       select(selectRouteNestedParams),
       map(params => {
         const createdAt = params['techCreatedAt'];
-        const isProvisional = this.router.url.split('/').pop() === 'provisional';
+        const lastTwoUrlParts = this.router.url.split('/').slice(-2)
 
-        if (isProvisional) {
+        function isProvisional() {
+          if (lastTwoUrlParts.includes('provisional')) {
+            return true
+          }
+          return false;
+        }
```

Comment:
```suggestion
       const isProvisional = () => lastTwoUrlParts.includes('provisional');
```
I think you can simplify the function like this.

Reply:
I wanted it to specifically look at the last two parts of the URL in case of any other potential side effects, but I couldn't exactly think of any off the top of my head